cmd/dep/ensure.go

```diff
@@ -162,7 +162,7 @@ func (cmd *ensureCommand) Run(ctx *dep.Ctx, args []string) error {
 		writeV = dep.VendorAlways
 	}
 
-	newLock := dep.LockFromInterface(solution)
+	newLock := dep.LockFromSolution(solution)
```

Comment:
🎉 I know this is a small change, but the name is so much more clear that this gave me a happy.